admin/qtx_import_export.php

```diff
@@ -229,6 +229,21 @@ function qtranxf_admin_section_import_export( $request_uri ) {
             'text' => sprintf( __( 'Use plugin %s to import data.', 'qtranslate' ), '<a href="https://wordpress.org/plugins/w2q-wpml-to-qtranslate/" target="_blank">W2Q: WPML to qTranslate</a>' )
         ) ) ?>
         <?php do_action( 'qtranslate_add_row_migrate' ) ?>
+        <tr id="qtranslate-import-slugs">
+            <th scope="row"><?php _e( 'Import from slugs', 'qtranslate' ) ?></th>
+            <td>
+                <label for="qtranslate_import_slugs"><input type="checkbox" name="qtranslate_import_slugs"
+                                                            id="qtranslate_import_slugs"
+                                                            value="1"/> <?php _e( 'Import options, post and term meta from legacy slugs (QTS). Attention, existing slugs will be deleted.', 'qtranslate' ); ?>
```

Comment:
Maybe we could hide the confirmation checkbox unless the first is checked.
We may add inline to the first input element something like `onclick="let x=jQuery('#qtranslate_import_slugs_confirm');if (jQuery(this).prop('checked')===true){x.parent().show();}else{x.prop('checked', false);x.parent().hide();}"`.

Reply:
Yes, or maybe better, make it disabled (grayed) and enable it when the first one is checked.

Reply:
agreed, implemented in 5ed9b2e7088b0ca2f408a5e49bf366bfd7d36baf